lib/controllers/DeviceController.ts

```diff
@@ -85,6 +85,30 @@ export class DeviceController extends CRUDController {
     };
   }
 
+  async update (request: KuzzleRequest) {
+    const deviceId = request.getId();
+    const body = request.getBody();
+    const device = await this.mGetDevice([{ deviceId }]);
```

Comment:
Nitpicking naming: here this represents many devices
```suggestion
    const devices = await this.mGetDevice([{ deviceId }]);
```